### ISSUE TITLE
PHP 5.3.3

### ISSUE THREAD

**Issue by mfrank410** [AuthorizeNet/sdk-php#207]
I can't get composer to download the dependencies. Our server is currently running 5.3.3 and upgrading at the moment will not work for us.

On the GitHub Code page, you list the requirements as PHP 5.3+. However, when I run composer to install the dependencies I get the following error messages:

Problem 1
    - goetas-webservices/xsd2php-runtime v0.2.3 requires php >=5.5 -> your PHP version (5.3.3) does not satisfy that requirement.
    - goetas-webservices/xsd2php-runtime v0.2.2 requires php >=5.5 -> your PHP version (5.3.3) does not satisfy that requirement.
    - Installation request for goetas-webservices/xsd2php-runtime ^0.2.2 -> satisfiable by goetas-webservices/xsd2php-runtime[v0.2.2, v0.2.3].

Since you have the requirements documented as PHP 5.3+, can you explain to me how I can get these dependencies installed?

Thank You,

**Comment by ddrght12345** [comment]
Actually, if you read the composer require, you'll see it states PHP >= 5.5
```json
{
  "require": {
  "php": ">=5.5", <-- Here
  "ext-curl": "*",
  "authorizenet/authorizenet": "1.9.2"
  }
}
```

**Comment by ddrght12345** [comment]
Also: This is a repo managed by authorize. You can not reasonably expect their documentation to match up to the actual code. Even when it's on the same page.

**Comment by mfrank410** [comment]
Haha, thanks. I am aware of the composer requirements. Regardless, I've decided to go the server upgrade route. I was initially going to just upgrade PHP to 5.6, but then realized the cloud server we are running has RHEL 5.5 (end of life is coming up soon). So long story short, I had to spin up a new server, configure it, and start migrating the web sites over. But at least now I am running PHP 5.6 and shouldn't have any issues installing the authorize.net libraries.

Does anyone know if you can still access the old authorize.net API documentation?

Thank You,

**Comment by ashtru** [comment]
PHP 5.5 is the minimum required for Authorize.Net PHP SDK.

@mfrank410 I am not clear by what you mean by old documentation. What do you want to access in the old authorize.net API documentation?

**Comment by clarkw** [comment]
Is there an earlier version of the SDK that I can use for PHP 5.3.22? I am currently stuck on that version.

**Comment by adavidw** [comment]
Hi @clarkw,

No chance you can upgrade? The 5.3 branch has been EOL since mid-2014, and 5.3.22 is over four years old now, which means it's [riddled with security vulnerabilities](https://www.cvedetails.com/vulnerability-list/vendor_id-74/product_id-128/version_id-147275/PHP-PHP-5.3.22.html).

PCI-DSS compliance is a requirement of anyone using a merchant account, and the 5.3 branch is not compliant because it no longer receives security fixes. Even there were still 5.3 security patches in development, your environment wouldn't be compliant since you haven't installed them. The only way to make a PCI-DSS compliant version of 5.3 would be to backport all subsequent security fixes from 5.4, 5.5, and 5.6.

What does your PCI-DSS QSA say?
